### DISCUSSION TITLE
Welcome to SAJeSolar Discussions!

### DISCUSSION THREAD

**Discussion by djansen1987** [djansen1987/SAJeSolar#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cabriltt** [comment]
Hi
I have just instaled a new SAJ h1 series in my home and i am trying to integrate in home assinstant with no luck.
I have a meter but i dont think i have a sec module.
After the configuration al the sensors have the state as unkown.
Can you help me?
Many thanks

**Reply by djansen1987** [reply]
Resuming this on: 
[https://github.com/djansen1987/SAJeSolar/discussions/10](https://github.com/djansen1987/SAJeSolar/discussions/10)
